modules/phase_field/include/ics/MultiSmoothSuperellipsoidIC.h

```diff
@@ -31,29 +31,50 @@ class MultiSmoothSuperellipsoidIC : public SmoothSuperellipsoidBaseIC
 
   virtual void initialSetup();
 
-
 protected:
   virtual void computeSuperellipsoidSemiaxes();
   virtual void computeSuperellipsoidCenters();
   virtual void computeSuperellipsoidExponents();
+  virtual Real overlapCheck(const Point & newcenter, const Real & _nc_as, const Real & _nc_bs, const Real & _nc_cs, const Real & _nc_ns);
 
+  unsigned int _numtries;
   unsigned int _numbub;
   Real _bubspac;
-
-  unsigned int _numtries;
   Real _exponent;
   Real _semiaxis_a;
   Real _semiaxis_b;
   Real _semiaxis_c;
   Real _semiaxis_a_variation;
   Real _semiaxis_b_variation;
   Real _semiaxis_c_variation;
+  Real _nv;
```

Comment:
I removed all of those variables except _nv, each of the others were only used once so they were not helping. Instead it just calls vector.size() in the IF statement arguments when needed for the others. Since _numbub_vec.size() was used numerous times in the IF arguments i left it assigned to _nv to reduce the number of vector.size() calls and keep the IF statement a little more compact.